modules/common/adapters/message_adapters.h

```diff
@@ -62,6 +63,8 @@ using PadAdapter = Adapter<control::PadMessage>;
 using PerceptionObstaclesAdapter = Adapter<perception::PerceptionObstacles>;
 using PlanningAdapter = Adapter<planning::ADCTrajectory>;
 using PointCloudAdapter = Adapter<::sensor_msgs::PointCloud2>;
+using ImageShortAdapter = Adapter<::sensor_msgs::Image>;
+using ImageLongAdapter = Adapter<::sensor_msgs::Image>;
```

Comment:
Simply define one ImageAdapter and use it for both.

Reply:
adapter is designed as a topic receiver instead of a sensor receiver, as is said by @siyangy

Reply:
Oops, you are right :)